### ISSUE TITLE
Add automatic link detection to video/series descriptions and text blocks

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#1074]
It's annoying if you have to select some text, copy, open new tab and paste. Things that are obviously links should be automatically turn into an `<a>` element. We miiight even render internal links to certain routes in a fancy way. But that's not that important.

There seem to be a few libraries for the detection part:
- https://www.npmjs.com/package/linkifyjs
- https://www.npmjs.com/package/linkify-it
